js/tji-datasets-explore.js

```diff
@@ -194,17 +196,19 @@ TJIGroupByDoughnutChart.prototype.create_legend = function(keys) {
 // * Constructor arguments:
 // *   chart_configs: array of chart config objects,
 // *                  e.g. [{type: 'bar', group_by:'year'}, ...]
-// *   charts_elt_id: id of HTML element to put charts in
-// *   filters_elt_id: id of HTML element to put filter checkboxes in
+// *   charts_elt_selector: id of HTML element to put charts in
+// *   filters_elt_selector: id of HTML element to put filter checkboxes in
 // *   chart_wrapper_template: HTML to wrap around each chart's canvas object
 // *   record_count_template: HTML for the "showing this man records" element
 // *       at the top, containing a "{count}" placeholder somewhere
 // *       (which TJIChartView will replace with the record count).
+// *
+// * Dependencies: TJIGroupByBarChart, TJIGroupByDoughnutChart
+// *       jQuery, lodash, auto-complete.min.js - https://github.com/Pixabay/JavaScript-autoComplete
 // ********************************************************************
 
 
-var TJIChartView = function(chart_configs, charts_elt_id, filters_elt_id,
-                            chart_wrapper_template, record_count_template){
+var TJIChartView = function(props){
```

Comment:
<3 <3

While we're doing the whole props thing, should we change the `TJIGroupBy*Chart` objects to do the same?

![props](https://user-images.githubusercontent.com/1421476/42423093-e195063e-82b9-11e8-9389-7e7fa944fbd8.gif)

Reply:
can do.